<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametryBadaniaOgolneDlg</class>
 <widget class="QWidget" name="ParametryBadaniaOgolneDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>451</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1200</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QGroupBox" name="gbPassword">
     <property name="title">
      <string>Hasło dostępu do parametrów pomiaru</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lhaslo">
        <property name="text">
         <string>Wprowadź hasło</string>
        </property>
        <property name="buddy">
         <cstring>password1</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="password1">
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lhaslo2">
        <property name="text">
         <string>Powtórz hasło</string>
        </property>
        <property name="buddy">
         <cstring>password2</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="password2">
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="pbOdblokuj">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Odblokuj</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbZmienHaslo">
          <property name="text">
           <string>Ustaw hasło</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Badanie</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="eNumerZlecenia">
        <property name="text">
         <string>Numer zlecenia</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>numerZlecenia</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="numerZlecenia"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lnumertestu">
        <property name="text">
         <string>Numer testu</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>numerTestu</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="losobaodpowiedzialna">
        <property name="text">
         <string>Osoba odpowiedzialna</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>osobaOdpowiedzialna</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="numerTestu"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="osobaOdpowiedzialna"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QGroupBox" name="gbKonfiguracjaFiltrow">
     <property name="title">
      <string>Konfiguracja filtrów</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="ldlugoscfali">
        <property name="text">
         <string>Użyj zestawu filtrów dla długości fali</string>
        </property>
        <property name="buddy">
         <cstring>dlugoscFali</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QComboBox" name="dlugoscFali">
          <property name="minimumSize">
           <size>
            <width>145</width>
            <height>0</height>
           </size>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::InsertAtBottom</enum>
          </property>
          <item>
           <property name="text">
            <string>880</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>655</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dlugoscFaliLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="lunumdlugoscfali">
          <property name="text">
           <string>[nm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Czas pomiędzy zmianami tłumienia filtra</string>
        </property>
        <property name="buddy">
         <cstring>czasPomiedzyZmianamifiltra</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLineEdit" name="czasPomiedzyZmianamifiltra"/>
        </item>
        <item>
         <widget class="QLabel" name="lunitczapomiedzytlumienia">
          <property name="text">
           <string>[s] (10-3600)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Zasilanie czujek</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbInsideSupply">
        <property name="text">
         <string>Z  zasilacza,  napięciem:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="napiecieZasilaniain"/>
        </item>
        <item>
         <widget class="QLabel" name="lunitvoltinside">
          <property name="text">
           <string>[V] (2-30)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rbCentralSupply">
        <property name="text">
         <string>Z centrali sygnalizacji pożarowej typu</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="typCentraliSygnalizacji"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lczasstabilizacjiczujki">
        <property name="text">
         <string>Czas stabilizacji czujki po włączeniu zasilania</string>
        </property>
        <property name="buddy">
         <cstring>czasStabilizacjiCzujki</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="czasStabilizacjiCzujki"/>
        </item>
        <item>
         <widget class="QLabel" name="lunitczasstabilizacji">
          <property name="text">
           <string>[s] (10-3600)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lczasstabilizacjiczujkireset">
        <property name="text">
         <string>Czas stabilizacji czujki po resecie</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLineEdit" name="czasStabilizacjipoResecie"/>
        </item>
        <item>
         <widget class="QLabel" name="eunitczasresetuczuji">
          <property name="text">
           <string>[s] (1-3600)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QGroupBox" name="gbSposobSygnalizacjiAlarmu">
     <property name="title">
      <string>Sposób sygnalizacji alarmu</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rbAlarmPrzekaznik">
        <property name="text">
         <string>Centrala lub (przekaźnik NO - normalnie otwarty)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="rbPrad">
          <property name="text">
           <string>Przekroczenie wartości prądu zasilania</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pradAlarmu"/>
        </item>
        <item>
         <widget class="QLabel" name="lunitCurrent">
          <property name="text">
           <string>mA (4-200)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="gbUwagi">
     <property name="title">
      <string>Uwagi</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextEdit" name="uwagi"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QDialogButtonBox" name="buttonBox_2"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>numerZlecenia</tabstop>
  <tabstop>numerTestu</tabstop>
  <tabstop>osobaOdpowiedzialna</tabstop>
  <tabstop>uwagi</tabstop>
  <tabstop>rbInsideSupply</tabstop>
  <tabstop>napiecieZasilaniain</tabstop>
  <tabstop>rbCentralSupply</tabstop>
  <tabstop>typCentraliSygnalizacji</tabstop>
  <tabstop>czasStabilizacjiCzujki</tabstop>
  <tabstop>rbAlarmPrzekaznik</tabstop>
  <tabstop>rbPrad</tabstop>
  <tabstop>pradAlarmu</tabstop>
  <tabstop>dlugoscFali</tabstop>
  <tabstop>czasPomiedzyZmianamifiltra</tabstop>
  <tabstop>password1</tabstop>
  <tabstop>password2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>accept()</slot>
  <slot>reject()</slot>
 </slots>
</ui>
